components/logged_in.jsx

```diff
@@ -65,11 +51,25 @@ export default class LoggedIn extends React.Component {
         // Listen for user
         UserStore.addChangeListener(this.onUserChanged);
 
-        // Listen for focussed tab/window state
+        // Listen for focused tab/window state
         window.addEventListener('focus', this.onFocusListener);
         window.addEventListener('blur', this.onBlurListener);
 
-        // ???
+        // Because current CSS requires the root tag to have specific stuff
+        $('#root').attr('class', 'channel-view');
+
+        // Device tracking setup
+        if (UserAgent.isIos()) {
+            $('body').addClass('ios');
+        } else if (UserAgent.isAndroid()) {
+            $('body').addClass('android');
+        }
+
+        if (!this.state.user) {
+            $('#root').attr('class', '');
```

Comment:
This is the actual fix; others are just rearranging the codes which move other logics from `LoggedIn`'s constructor to its componentDidMount.

Reply:
Are you meaning to remove all class values or just `android` or `channel-view`? Because jQuery does have a method to [remove classes by name](https://api.jquery.com/removeClass/).

Reply:
That's for all class values.